### ISSUE TITLE
Tons of error messages in log output

### ISSUE THREAD

**Issue by suzukieng** [idnow/de.idnow.ios#41]
SDK version: 3.9.0

During a video identification, the SDK logs thousands of lines like these:

```
2018-06-21 12:37:51.564350+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.581391+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.601944+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.623017+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.644363+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.665710+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.687445+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.708437+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.729776+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
2018-06-21 12:37:51.751494+0200 HIN VideoIdent[295:15152] 55: EXCEPTION (-1): ""
```

**Comment by agrison** [comment]
We have the exact same problem.

**Comment by phu-idnow** [comment]
Hi @suzukieng , @agrison ,

This is caused by a bug in iOS 11. Icelink reported and fixed it in Icelink3. Our side updated it also and will ship in next release.
reference: https://support.frozenmountain.com/hc/en-us/community/posts/115003026754/comments/115001357813 

Thanks

**Comment by sasha-tomorrow** [comment]
Still not fixed in 3.15.0
```
2019-01-21 12:43:36.344376+0100 Debug[9092:731450] TIC Read Status [66:0x0]: 1:57
2019-01-21 12:43:36.344432+0100 Debug[9092:731450] TIC Read Status [66:0x0]: 1:57
2019-01-21 12:43:36.352230+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.373873+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.394939+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.416350+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.437947+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.458694+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.480492+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.501715+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.522551+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.544691+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.565676+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.586718+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.608946+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.629462+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.651469+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.672788+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.693452+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.694102+0100 Debug[9092:725805] POST change foreground state: https://api.test.idnow.de/api/v1/solarisbankvideoidentsandbox/identifications/TST-JJPUD/appEnteredBackground
2019-01-21 12:43:36.714936+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.728100+0100 Debug[9092:725805] [Snapshotting] Snapshotting a view (0x1090ab800, UIKeyboardImpl) that has not been rendered at least once requires afterScreenUpdates:YES.
2019-01-21 12:43:36.736232+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.757200+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.778191+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.800197+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.821636+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.843919+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.864175+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.879436+0100 Debug[9092:725805] POST foreground state -> background: successful
2019-01-21 12:43:36.885780+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.907382+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.929148+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.950160+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.971833+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:36.992746+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.013856+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.035300+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.056878+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.078126+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.099162+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.120643+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.142185+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.163000+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.184358+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.205751+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.227143+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.249365+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.271041+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.291907+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.312623+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.333980+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.355153+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.376449+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.397546+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.419013+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.440356+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.462273+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.483088+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.504346+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.527146+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.548418+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.568431+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.591150+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.611728+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.632312+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.655541+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.676208+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.696475+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.717856+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.739060+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.760396+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.782075+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.803099+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.824442+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.845828+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.866835+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.888341+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.909636+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.932591+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
2019-01-21 12:43:37.951627+0100 Debug[9092:729946] 55: EXCEPTION (-1): ""
```

**Comment by danielkiedrowski** [comment]
Still not fixed in 3.21.0. :(

**Comment by aare-undo-idnow** [comment]
Probably fixed 🥳
